backend/src/Repository/PlantRepository.php

```diff
@@ -30,6 +30,28 @@ public function deletePlant(Plant $plant): void {
         $this->_em->flush();
     }
 
+    public function convertDate(Plant $plant): Plant {
```

Comment:
If i understand it right, you are not converting or?
Isn't it something like settingCarePeriod or so?

Reply:
You're right, could called it setTimeLeft or something like that 👍